project/JdkOptions.scala

```diff
@@ -57,14 +57,8 @@ object JdkOptions extends AutoPlugin {
       targetSystemJdk,
       jdk8home,
       fullJavaHomes,
-      Seq(if (scalaVersion.startsWith("3.")) "-Xtarget:8" else "-target:jvm-1.8"),
-      // '-release 8' is not enough, for some reason we need the 8 rt.jar
-      // explicitly. To test whether this has the desired effect, compile
-      // pekko-remote and check the invocation of 'ByteBuffer.clear()' in
-      // EnvelopeBuffer.class with 'javap -c': it should refer to
-      // ""java/nio/ByteBuffer.clear:()Ljava/nio/Buffer" and not
-      // "java/nio/ByteBuffer.clear:()Ljava/nio/ByteBuffer". Issue #27079
-      (java8home: File) => Seq("-release", "8", "-javabootclasspath", java8home + "/jre/lib/rt.jar"))
```

Comment:
The removal of `-javabootclasspath` breaks building with JDK 11 for me, also reproducible in Docker containers. I know I investigated before, if it could be removed and I ran into issues before.

I wonder how it works on Github CI and other machines.

Reply:
interesting, seems to work on my machine - what does it break on for you?

Reply:
@jrudolph we have a release candidate (1.0.1-RC1) at https://repository.apache.org/content/groups/staging/org/apache/pekko/

Would you be able to see if there is anything missing in the jars?

The RCs have been built on my Mac. I use sdkman to manage my JDK installs.

The CI builds all seem fine.

I have no objection to putting back the `javabootclasspath` if it makes the build work for you. In the short term, I would be worried that there is something up with RC and the 1.0.0 release.

Reply:
I haven't looked at this conversation, but FYI there was a bug fix to prefer javabootclasspath, someone needed to compile against a specific version of rt.jar. This PR https://github.com/scala/scala/pull/10336

Reply:
The RC1 pekko-remote jar: I ran the `javap -c` on the EnvelopeBuffer class and got

```
      9: invokevirtual #121                // Method java/nio/ByteBuffer.clear:()Ljava/nio/Buffer;
```

Which I think means that the RC1 jar is not affected by Issue #27079

Reply:
> I haven't looked at this conversation, but FYI there was a bug fix to prefer javabootclasspath, someone needed to compile against a specific version of rt.jar. This PR [scala/scala#10336](https://github.com/scala/scala/pull/10336)

Thanks @som-snytt - the Pekko build uses Scala 2.13.11 so we might be benefitting from the https://github.com/scala/scala/pull/10336 change.